.github/wiki/Getting-Started.md

```diff
@@ -131,20 +131,27 @@ $global:lastRepository = $null
 function Check-DirectoryForNewRepository {
     $currentRepository = git rev-parse --show-toplevel 2>$null
     if ($currentRepository -and ($currentRepository -ne $global:lastRepository)) {
-        onefetch
+        if ([Console]::OutputEncoding -ne [System.Text.Encoding]::UTF8) {
+            [Console]::OutputEncoding = [System.Text.Encoding]::UTF8
+        }
+        onefetch | Write-Host
     }
     $global:lastRepository = $currentRepository
 }
 
 function Set-Location {
     param (
-        [string]$path
+        [string]$Path,
+        [string]$LiteralPath
     )
 
-    # Use the default Set-Location to change the directory
-    Microsoft.PowerShell.Management\Set-Location -Path $path
+    if ($LiteralPath) {
+        Microsoft.PowerShell.Management\Set-Location -LiteralPath $LiteralPath
+    }
+    else {
+        Microsoft.PowerShell.Management\Set-Location -Path $Path
+    }
```

Comment:
Instead of explicitly capturing certain known arguments, perhaps it would be better to just pass *all* arguments to the built-in. Look up `@args`.

Reply:
Just this directly?

```pwsh
Microsoft.PowerShell.Management\Set-Location @args
```

Reply:
Yeah, I think so, unless you see any problems with that. It should pass *every* parameter to the builtin AFAIK.

Reply:
In Python terms, it's the difference between doing this:
```python
def my_print(value, end='\n'):
    print(value, end=end)
```

and this:

```python
def my_print(*args, **kwargs):
    print(*args, **kwargs)
```

AFAIK

Reply:
Yep, tested it out with different cases, didn't cause any issues, we can go ahead with `@args`